es-models/case_set/settings.yaml

```diff
@@ -0,0 +1,2 @@
+index.mapping.nested_fields.limit: 100
```

Comment:
There is no nesting here, don't really need `nested_fields.limit`, ES default will work